src/mds/MDSRank.cc

```diff
@@ -249,6 +250,7 @@ class C_Flush_Journal : public MDSInternalContext {
 
   MDCache *mdcache;
   MDLog *mdlog;
+  SegmentBoundary* sb = nullptr;
```

Comment:
minor nit: this only needs the seq number to pass to trim_to() so just include that rather than using SegmentBoundary?

Reply:
sure, changed